### PR TITLE
Disable components when data processing is underway

### PR DESCRIPTION
**Pull request by cosa65**
# Background
#### Link to issue 
https://biomage.atlassian.net/browse/BIOMAGE-498

#### Link to staging deployment URL 
https://ui-martin-ui144-api62.scp-staging.biomage.net/

#### Links to any Pull Requests related to this
https://github.com/biomage-ltd/api/pull/62

#### Anything else the reviewers should know about the changes here

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [x] Unit tests written
- [x] Tested locally with Inframock
- [ ] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [ ] Relevant Github READMEs updated
- [ ] Relevant wiki pages created/updated

### Approvers
- [ ] Approved by a member of the core engineering team
- [ ] (UX changes) Approved by vickymorrison (this is her username, tag her if you need approval)

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [ ] Photo of a cute animal attached to this PR

### CODE REVIEW

**Comment by ivababukova** [comment]
why is this PR a WIP? What is missing?

**Comment by cosa65** [comment]
> why is this PR a WIP? What is missing?

Now it is no longer a WIP! I needed to wait and make sure it matched the pipeline status object we receive from the api.

File: src/components/ContentWrapper.jsx [hunk 1]
```diff
@@ -43,6 +44,15 @@ const ContentWrapper = (props) => {
     status: pipelineStatus,
   } = useSelector((state) => state.experimentSettings.pipelineStatus);
 
+  console.log('pipelineStatusDebug');
+  console.log(pipelineStatus);
```
Comment (ivababukova) [inline src/components/ContentWrapper.jsx]: no need for this

**Comment by ivababukova** [comment]
looks good code-wise. Have you tested in staging?

**Comment by marcellp** [comment]
Would be happy to approve if it works end-to-end, as @ivababukova said.

**Review by ivababukova — APPROVED** [review]

**Comment by cosa65** [comment]
In staging it seems to be detecting some kind of error, so it is locking the ui to data processing, I tried redeploying and it still shows the issue.

This is something I didn't run into when testing locally, (but locally I am using the local runner).

On lens I am seeing this:
image

**Comment by xverges** [comment]
> In staging it seems to be detecting some kind of error,

Maybe we should modify https://github.com/biomage-ltd/api/blob/master/src/utils/schema-validator.js#L40 to log the request that failed to validate before throwing.

**Pull request merged**